streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ResourceServiceImpl.java

```diff
@@ -204,13 +205,18 @@ public void deleteResource(Resource resource) {
     Resource findResource = getById(resource.getId());
     checkOrElseAlert(findResource);
 
-    FsOperator.lfs()
-        .delete(
-            String.format(
-                "%s/%d/%s",
-                Workspace.local().APP_UPLOADS(),
-                findResource.getTeamId(),
-                findResource.getResourceName()));
+    String filePath =
+        String.format(
+            "%s/%d/%s",
+            Workspace.local().APP_UPLOADS(),
+            findResource.getTeamId(),
+            findResource.getResourceName());
+
+    if (!new File(filePath).exists() && StringUtils.isNotBlank(findResource.getFilePath())) {
```

Comment:
We better use FilesUtils.exists(path...)

Reply:
Ok, thanks for reviewing the code